tests/FormatNextGenQuery.test.ts

```diff
@@ -0,0 +1,35 @@
+import { formatFedQueryForNextGen } from "../utils/httpUtils";
+
+describe("FormatFedQuery:", () => {
+  describe("formatFedQuery:", () => {
+    it("Removes fed prefix from the top level query", () => {
+      expect(formatFedQueryForNextGen("query fedConsensusGenome")).toBe("query consensusGenome");
+    });
+
+    // it("It does not remove fed from lower in the query", () => {
+    //   expect(
+    //     formatFedQueryForNextGen({
+    //       param1: 123,
```

Comment:
I would like to complete these two pieces of functionality for this function but I thought it would be good to get to main first so that everyone can use this connection code.